consensus/types/src/attestation.rs

```diff
@@ -601,12 +601,12 @@ mod tests {
         let attestation_data = size_of::<AttestationData>();
         let signature = size_of::<AggregateSignature>();
 
-        assert_eq!(aggregation_bits, 152);
+        assert_eq!(aggregation_bits, 144);
```

Comment:
I have no idea why this would change because of a dependency update.
cc @paulhauner

Reply:
I think smallvec sometimes implements changes to its layout and size between versions, so this is fine